dbt_project.yml

```
@@ -155,3 +162,6 @@ models:
     br_rj_riodejaneiro_rdo:
       +materialized: view
       +schema: br_rj_riodejaneiro_rdo
+    br_rj_riodejaneiro_bilhetagem:
+      +materialized: view
```

Deixar apenas as tabelas de produção no dataset e colocar como padrão o modelo incremental

```suggestion
      +materialized: incremental
```